Remotion/Web/Development.WebTesting/CoypuElementScopeFillInWithAndSendKeysExtensions.cs

```diff
@@ -105,11 +106,26 @@ private static void SetValueUsingSendKeys ([NotNull] this ElementScope scope, [N
       value = value.Replace("\r", "");
 
       var clearTextBoxWithoutTriggeringPostBack = Keys.Control + "a" + Keys.Control + Keys.Delete;
-      value = clearTextBoxWithoutTriggeringPostBack + value;
 
-      s_log.DebugFormat("FillInWith without triggering PostBack on clear: '{0}'.", value);
-
-      scope.SendKeys(value);
+      if (scope.Browser.IsFirefox())
+      {
+        //GeckoDriver cannot handle inputting the value after receiving the delete command, IF both were sent in the same SendKeys.
+        //This occurs after Firefox v99 and would lead to an empty field.
```

Comment:
```suggestion
        // GeckoDriver cannot handle inputting the value after receiving the delete command, iF both were sent in the same SendKeys.
        // This occurs after Firefox v99 and would lead to an empty field.
```